parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -76,6 +76,9 @@ if [[ "${SHOULD_CONFIGURE_CUSTOM_CA_TRUST}" == "true" ]]; then
 fi
 
 if [[ -n "${OUTBOUND_COMMAND}" ]]; then
+    if [[ -n "${PROXY_VARS}" ]]; then
+        eval $PROXY_VARS
```

Comment:
kinda annoying to form this export string properly without eval...

don't love eval, but pretty sure we sanitize this reasonably well in RP

and if we don't...congratulations, you're paying to hack your own VMs via shell script injection